HACKING.md

```diff
@@ -160,6 +160,38 @@ your charm to a controller using that version of Juju. For example, with microk8
 We rely on automation to [update charm pins](.github/actions/update-charm-pins/) of
 a bunch of charms that use the operator framework. The script can be run locally too.
 
+# Contributing
+
+Changes are proposed as [pull requests on GitHub](https://github.com/canonical/operator/pulls).
+
+Pull requests should have a short title that follows the
+[conventional commit style](https://www.conventionalcommits.org/en/) using one of these types:
+
+* chore
+* ci
+* docs
+* feat
+* fix
+* perf
+* refactor
```

Comment:
I think "refactor" is a "chore". Thoughts on getting rid of it? Or maybe we should go the other way -- what chores are there that *aren't* refactoring?

Also, what about dropping "perf"? I kinda think we'll almost never need this in Ops. But maybe it doesn't hurt, even as a signal that we are not unconcerned about perf.

Reply:
For the record, I dropped "build" and "style" from the [Angular list that conventionalcommits.com recommends](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type).

If we get rid of both perf I would put a change that's purely performance improvements under refactor, so I'm not sure where that would go if we get rid of both. It doesn't seem like a chore, and maybe that is going too... I like keeping it, even if seldom-used, as a signal as you describe.

Other chores: dependency bumps, the release PRs. Looking back over the last few months, there are some that are "internal tooling" (they could probably have been categorised into one of the others). I'm +0 for keeping refactor and chore.

Reply:
Okay, sounds fine with me -- let's keep 'em. Being a little more specific doesn't hurt. As long as this list doesn't get too huge.